### ISSUE TITLE
Subdirectories of package imports do not load as expected for github dependencies.

### ISSUE THREAD

**Issue by josephperrott** [stackblitz/core#149]
I have created a reproduction [here](https://stackblitz.com/edit/github-subdirectory-loading?file=app/app.module.ts).

If you change from `@angular/materal` imported from github (done by entering `https://github.com/angular/material2-builds` as the package name) to `@angular/material` as the package name, it works as expected.

Not sure if that makes sense, let me know if I need to provide a better explanation.

**Comment by EricSimons** [comment]
@josephperrott you're absolutely correct- this is actually [the main limitation](https://github.com/stackblitz/core/issues/36#issuecomment-332681159) of the Github packages feature right now. I'll prioritize some time to get a fix in for this; shooting to have on prod by EOD :)

**Comment by josephperrott** [comment]
Awesome, thanks!  Let me know if there is anything I can do to help

**Comment by EricSimons** [comment]
@josephperrott this is now live on prod! You might want to uninstall then reinstall any github deps in projects that were breaking before to ensure everything works properly. The github resolution logic now has parity with the NPM versions, including:

- Nested package resolution
- Node-style folder & file resolution
- SASS, LESS, and CSS imports

Here's a demo of Angular Material running the latest build from Github & using a custom SASS material theme: https://stackblitz.com/edit/angular-m9axcs?file=theme.scss

Lmk if everything looks good on your end!

**Comment by ThomasBurleson** [comment]
@EricSimons - can you describe the nested-package resolution issue better? My apologies but I do not understand the issue reported by @josephperrott .

**Comment by josephperrott** [comment]
@EricSimons Works great! thank you so much for the quick turnaround on this!

@ThomasBurleson there were issues initially with being able to import something like:
`@angular/material/autocomplete`, but `@angular/material` was working as expected

**Comment by ThomasBurleson** [comment]
@josephperrott - I understand now what you mean by nested packages.  Thx.

**Comment by ThomasBurleson** [comment]
@josephperrott - can we close this issue here ?
